tests/e2e/utils/pageNavigation.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const { HomePage } = require('../pages/homePage');
+const { PreferencePage } = require('../pages/prefPage');
+
+const pages = {
+  dashboard: `${process.env.TEST_BASE_URL}/user/dashboard`,
+  preference: `${process.env.TEST_BASE_URL}/user/preferences`,
+  breach: `${process.env.TEST_BASE_URL}/user/breaches`,
+  security: `${process.env.TEST_BASE_URL}/user/security-tips`
```

Comment:
Q: Do these need `TEST_BASE_URL` if we already have that set as a default in the Playwright `baseURL` config? (see playwright.config.js#L61)